edge-hub/README.md

```diff
@@ -13,18 +13,63 @@ This project contains the Edge Hub.
 
 
 ## How to debug Edge hub using Visual Studio
-1. Set environment variable `EdgeModuleHubServerCertificateFile` to the path of a SSL Certificate file (e.g. C:\edgeDevice.pfx); for debug purpose, you can [create](https://docs.microsoft.com/en-us/azure/cloud-services/cloud-services-certs-create#create-a-new-self-signed-certificate) and use a self-signed certificate.  Remember to restart Visual Studio to take effect.
-2. Update following values in appsettings_hub.json in Microsoft.Azure.Devices.Edge.Hub.Service project. 
-  * `IotHubConnectionString` - Edge Hub module connection string; set it as edge device connection string appending with "ModuleId=$edgeHub".
+1. Create an IOT edgeHub module connection string.
+Create a Visual studio project following process here: https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-portal-csharp-module-twin-getstarted Section Update the module twin using .NET device SDK
```

Comment:
Since you already provided the complete code snippet, is it clear to state all steps here instead of having a pointer pointing to module twin example?

Reply:
On your first comment: I think the format of git hub is not as flexible to describe tutorial. It also has the advantage of keeping things in one place. So I'll rather keep the link.
On the second comment, yes, I will add 1-2 sentences.